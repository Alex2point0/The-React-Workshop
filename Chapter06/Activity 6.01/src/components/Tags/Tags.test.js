import React, { useState } from "react";
import { render, cleanup, fireEvent } from "@testing-library/react";
import { Tags } from "./index";

afterEach(cleanup);

const tags = ["black", "chocolate", "milk", "froth", "water"];

it("renders listing of products", () => {
  const { asFragment } = render(
    <Tags tags={tags} selectedTag={null} handleTags={() => {}} />
  );
  expect(asFragment()).toMatchSnapshot();
});
