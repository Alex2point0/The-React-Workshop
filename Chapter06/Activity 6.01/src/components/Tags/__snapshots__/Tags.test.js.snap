// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders listing of products 1`] = `
<DocumentFragment>
  <div>
    Select Filter
    <button
      class="tag "
    >
      black
    </button>
    <button
      class="tag "
    >
      chocolate
    </button>
    <button
      class="tag "
    >
      milk
    </button>
    <button
      class="tag "
    >
      froth
    </button>
    <button
      class="tag "
    >
      water
    </button>
  </div>
</DocumentFragment>
`;
